include/spdlog/details/log_msg-inl.h

```diff
@@ -22,15 +22,36 @@ SPDLOG_INLINE log_msg::log_msg(spdlog::log_clock::time_point log_time, spdlog::s
 #endif
     , source(loc)
     , payload(msg)
+#ifndef SPDLOG_NO_STRUCTURED_SPDLOG
+    , field_data(const_cast<Field *>(fields))
+    , field_data_count(field_count)
+    , context_field_data(threadlocal_context_head())
+#endif
+{
+#ifdef SPDLOG_NO_STRUCTURED_SPDLOG
+    (void) fields;
+    (void) field_count;
+#endif
```

Comment:
Is this to prevent compiler warnings re unused args? If so, perhaps a comment.
```suggestion
#ifdef SPDLOG_NO_STRUCTURED_SPDLOG
    // Prevent unused argument warning
    (void) fields;
    (void) field_count;
#endif
```